flask_appbuilder/models/sqla/__init__.py

```diff
@@ -46,7 +45,7 @@ def get_tables_for_bind(self, bind=None):
         return result
 
 
-class ModelDeclarativeMeta(_BoundDeclarativeMeta):
+class ModelDeclarativeMeta(DefaultMeta):
```

Comment:
Can you explain this change?

Reply:
> _BoundDeclarativeMeta is renamed to DefaultMeta; the old name will be removed in 3.0

see https://github.com/mitsuhiko/flask-sqlalchemy/blob/50944e77522d4aa005fc3c833b5a2042280686d3/CHANGES.rst#version-230
and
mitsuhiko/flask-sqlalchemy#546